v2/util/docker-cleanup.sh

```diff
@@ -1,17 +1,27 @@
 #!/bin/bash
 
-# this exists because trying to escape bash commands in a sytemd unit is super sadtimes
+# NOTE: this script will need to be sudo'ed for access to /var/lib/docker
```

Comment:
yes. I think standardizing as `core` is probably the right way to go. But if it starts to get too messy, we can run it as root if we need to.